src/components/hmi_message_handler/src/messagebroker_adapter.cc

```diff
@@ -116,6 +116,7 @@ void MessageBrokerAdapter::SubscribeTo() {
   MessageBrokerController::subscribeTo("VR.OnLanguageChange");
   MessageBrokerController::subscribeTo("TTS.OnLanguageChange");
   MessageBrokerController::subscribeTo("VehicleInfo.OnVehicleData");
+  MessageBrokerController::subscribeTo("VehicleInfo.OnReverseAppsAllowing");
```

Comment:
do we have VehicleInfo.OnReverseAppsAllowing?

Reply:
@LuxoftAKutsan Should this have been removed?